### ISSUE TITLE
Error converting summary statistics

### ISSUE THREAD

**Issue by gnarw** [bulik/ldsc#104]
I´m getting errors from munge_sumstats
I have recently updated some of the python packages, but I´m using python 2.7
The log with the error is not helping me finding the issue. (Attached file).
[iss.txt](https://github.com/bulik/ldsc/files/1719701/iss.txt)

Any ideas ?

**Comment by rkwalters** [comment]
Hello,
From previous of that error (here <https://github.com/bulik/ldsc/issues/93> and here <https://groups.google.com/d/msg/ldsc_users/9Mxw4vAB2l8/Muc_vtmYAwAJ>) is appears to be caused by changes in pandas 0.21. The current solution is to revert pandas to e.g. version 0.20.1.

@tpoterba: Can we look at whether this is fixable to support pandas 0.21 without breaking backwards compatibility?

Cheers,
Raymond

**Comment by tpoterba** [comment]
I don't think this makes much sense, since it'll just break in the next version. I think the current plan is to release a conda environment with the known passing versions (which I think is pandas 0.17).

**Comment by gnarw** [comment]
Reverting to pandas 0.18.1 solved this issue.
I had to revert to 0.18.1, else there were also problems when estimating the genetic correlation.
